algoliasearch/src/main/java/com/algolia/search/clients/InsightsConfig.java

```diff
@@ -0,0 +1,31 @@
+package com.algolia.search.clients;
+
+import com.algolia.search.models.CallType;
+import com.algolia.search.transport.StatefulHost;
+import java.time.LocalDate;
+import java.time.ZoneOffset;
+import java.util.Collections;
+import java.util.EnumSet;
+import java.util.List;
+
+@SuppressWarnings("WeakerAccess")
+public class InsightsConfig extends AlgoliaConfig {
+
+  public InsightsConfig(String applicationID, String apiKey) {
+    this(applicationID, apiKey, "us");
```

Comment:
Only `us`?

Reply:
`US` is the default one at the moment isn't it? 
That's why we provide another constructor to override it, if it's not the good one.

Reply:
👌